### PR TITLE
Update franken_errata.json

### PR DESCRIPTION
**Pull request by newnew1450**
Change Imperia to include Edict. 
Update some other DS components.

### CODE REVIEW

File: src/main/resources/data/franken_errata/franken_errata.json [hunk 1]
```diff
@@ -270,6 +270,18 @@
         "ItemId": "jolnar",
         "Undraftable": true
     },
+    {
```
Comment (jamesrudolph) [inline src/main/resources/data/franken_errata/franken_errata.json:273]: @newnew1450 which ruleset are you following that forces edict & imperia together? The one I know of from TTS/TTPG have them separated

Comment (newnew1450) [reply src/main/resources/data/franken_errata/franken_errata.json:273]: Oh I misremembered. Imperia gains the Commander, not Edict. I used this: https://docs.google.com/spreadsheets/d/1GWlcW1ue4oq-KwQCBBem8zF1LPX6a6QSxkP20jLmVu0/edit#gid=0

Comment (newnew1450) [reply src/main/resources/data/franken_errata/franken_errata.json:273]: @jamesrudolph how do I correct the update?

Comment (newnew1450) [reply src/main/resources/data/franken_errata/franken_errata.json:273]: Updated the change to Commander instead of Edict.

Comment (jamesrudolph) [reply src/main/resources/data/franken_errata/franken_errata.json:273]: I don't have access to that Google sheet, but I'll trust you lol

**Pull request merged**